packages/types/src/primitive/U8.ts

```diff
@@ -12,7 +12,7 @@ import UInt from '../codec/UInt';
  * An 8-bit unsigned integer
  */
 export default class U8 extends UInt {
-  constructor (value?: AnyNumber) {
-    super(value, 8);
+  constructor (registry: Registry, value?: AnyNumber) {
```

Comment:
Overall I'd suggest `constructor ({ registry }: { registry: Registry })`. With this signature it'd be extensible to pass ss58 id to Address and metadata to Event&Method

Reply:
The metadata ref. will probably be added to the registry, that is the idea, but I'm not there yet, so dunno. 

I started off with a `context: { registry: Registry }`, but that creates untold messes inside the actual registry where the values are passed though to `createType`. So, the calling context here for any type-related stuff will be the registry, and nothing more. (Which can be extended as the need arrises)